release-notes.txt

```diff
@@ -510,7 +480,7 @@ Fixed Issues from the last release (2019.04)
 
 Acknowledgements
 ================
-We would like to thank all companies (vendors) that provided us with (their)
+We would like to thank all companies that provided us with
```

Comment:
Now the formatting of this paragraph is off

Reply:
I didn't think it was that important.  All fixed.